layouts/_partials/blocks/premium-newsletter.html

```diff
@@ -1,9 +1,14 @@
 <section class="@container mb-6 w-full bg-g-9 pt-12">
   {{ $maxWidth := .Param "max-width" }}
-  {{ $maxWidthClass := cond (eq $maxWidth "prose") "max-w-prose" (print "max-w-[" $maxWidth "]") }}
+  {{ $maxWidthClass := cond (eq $maxWidth "prose") "max-w-prose" "max-w-[var(--spl-max)]" }}
```

Comment:
If you're using Tailwind 4, it should let you shortcut that as `max-w-(--spl-max)` with fewer brackets.  https://tailwindcss.com/docs/max-width#using-a-custom-value